webapp/src/components/sidebar_right/gitlab_items.tsx

```diff
@@ -194,7 +193,7 @@ function GitlabItems({item, theme}: GitlabItemsProps) {
                 {item.action_name && (
                     <>
                         <div>{item.updated_at && `Updated ${formatTimeSince(item.updated_at)} ago.`}</div>
-                        {notificationReasons[item.action_name]}
+                        {item.action_name == ACTION_NAME_MEMBER_ACCESS_REQUESTED && <a href={item.author.web_url}>{item.author.name}</a>} {notificationReasons[item.action_name]}
```

Comment:
I think this may be easier to read like this:

```js
const includeNotificationAuthor = item.action_name == ACTION_NAME_MEMBER_ACCESS_REQUESTED; 
```

```suggestion
        {includeNotificationAuthor && <a href={item.author.web_url}>{item.author.name}</a>} 
        {notificationReasons[item.action_name]}
```

Notice the separate lines as well